content/docs/user-guide/platforms/android/_index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Android Support for Open 3D Engine (O3DE)
+linkTitle: Android
+description: An overview of Open 3D Engine support for development for Android.
+weight: 200
+---
+
+O3DE projects for Android relies on the gradle build system to build the final APK required to launch and run O3DE projects. The gradle build scripts themselves uses cmake as a subset of the build process to compile the native C++ code, while gradle will also handle the build of the android application (java) and construction and package creation of the final APK and assets. 
+
+Because of this android projects require a custom script to generate the project beyond what cmake does by itself.
+
+## **Prerequisite Software and Packages**
+
+### **Java Development Kit (JDK)**
+The latest version of the Java Development Kit is needed to both compile the android java source for the project as well as to create a Java Key Storage (JKS) files used by the Android SDK to sign the APK.
+
+### **Gradle**
+[Gradle](https://gradle.org/install/) (version 6.7.1 or newer) is the build system designed to build and create Android applications.
+
+### **Android SDK** 
+The Android SDK will contain the android libraries, packages, ndk, tools, etc that are needed to build the Android App for an O3DE project. 
+
+### **CMake**
+[CMake](https://cmake.org/download/) project generator (version 3.21 or newer) is needed for O3DE for all native C++ builds, regardless of the platform. 
+
+### **Ninja Build System**
+The [ninja](https://ninja-build.org/) build system is used by CMake to build the underlying native C++ code for the O3DE project.
+
+## **Optional Software**
+
+### **Android Studio**
+Android Studio is the IDE commonly used in conjunction with Gradle to develop and build Android Apps. While Android Studio itself isn't absolutely necessary, it provides the mechanism to setup and download the required Android SDK for the build process. While its possible to just download the command line tools by itself, the current O3DE script that generates the Android Gradle project does not yet support this method. Download the latest version from [https://developer.android.com/studio](https://developer.android.com/studio) and run through the installation process.
+
+
+## **Setup Requirements**
+
+### **Setup the Android SDK**
+The Android SDK will be configured as part of the intial Android Studio experience. When you start Android Studio and have not setup the Android SDK yet, it will guide you through the steps to download and install it and at least one SDK platform.
```

Comment:
```suggestion
You can set up the Android SDK through Android Studio or Android command line tools. 
- If you're using Android Studio, you can install Android SDK during the initial Android Studio installation or from within Android Studio, if it's already installed. Then, Android Studio will guide you through the steps to download and install at least one SDK platform.
- If you're using Android command line tools, you must place the command line tool package in a subfolder called `tools` under the root of the Android SDK folder. 
```

Reply:
how about:

You can set up the Android SDK through Android Studio. You can install Android SDK during the initial launch Android Studio. Android Studio will guide you through the steps to download and install at least one SDK platform.

Reply:
Ah that's right. I made these suggestions before you confirmed that command line tools are not working at this time. Let's go with yours!